src/lib/dataset/helpers.ts

```diff
@@ -92,3 +92,70 @@ export function addLocalUniquesToDataset(dataset: DataSet) {
   }));
   return { ...dataset, headers: newHeaders };
 }
+
+/**
+ * uniqueStatsDataset is a Dataset with two columns:
+ *   - the inspected column
+ *   - the `count` column
+ *
+ * Example:
+ * uniqueStatsDataset = {
+ *   headers: [{name: 'City'}, {name: 'count'}]
+ *   data: [
+ *     ['Paris', 10],
+ *     ['Lyon', 1]
+ *     ['Marseille', 2]
+ *   ]
+ * }
+ * The output will be:
+ * ['City', {
+ *   Paris:  {value: 'Paris', count: 10},
+ *   Lyon:  {value: 'Lyon', count: 1},
+ *   Marseille:  {value: 'Marseille', count: 2},
+ * }]
+ */
+export function _prepareColumnStats(uniqueStatsDataset: DataSet): [string, ColumnValueStat[]] {
+  const columns: string[] = uniqueStatsDataset.headers.map(e => e.name);
+  const indexOfCount: number = columns.indexOf('count'); // equal to 1 in the example given above
```

Comment:
Are column names hardcoded? What if the original column name was `count`?

Besides, why doing `1 - indexOfCount` to compute `indexOfColumn` ? If something, shouldn't it rather be `indexOfCount - 1`?